packages/ketcher-react/src/script/ui/views/toolbars/TopToolbar/TopToolbar.tsx

```diff
@@ -87,8 +97,12 @@ const TopToolbar = (props: Props) => {
 
       <Group className={classes.meta}>
         <Item id="settings" />
-        <Item id="help" className={classes.metaInfo} />
-        <Item id="about" className={classes.metaInfo} />
+        {width && width >= 790 ? (
```

Comment:
Please move to constants

Reply:
fixed in [move media sizes to constants](https://github.com/epam/ketcher/pull/323/commits/d9abe72d289ea5e2293030fbaa0060c266759e0c)